packages/mobile/src/screens/track-screen/DownloadSection.tsx

```diff
@@ -0,0 +1,181 @@
+import { useCallback, useRef, useState } from 'react'
+
+import {
+  DownloadQuality,
+  cacheTracksSelectors,
+  useCurrentStems
+} from '@audius/common'
+import type { ID, CommonState } from '@audius/common'
+import { Animated, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import { IconReceive, IconCaretDown } from '@audius/harmony-native'
+import { Button, SegmentedControl, Text } from 'app/components/core'
+import { Expandable, useExpandable } from 'app/components/expandable'
+import { makeStyles } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useThemeColors } from 'app/utils/theme'
+
+import { DownloadRow } from './DownloadRow'
+
+const { getTrack } = cacheTracksSelectors
+
+const ORIGINAL_TRACK_INDEX = 1
+const STEM_INDEX_OFFSET_WITHOUT_ORIGINAL_TRACK = 1
+const STEM_INDEX_OFFSET_WITH_ORIGINAL_TRACK = 2
+
+const messages = {
+  title: 'Stems & Downloads',
+  choose: 'Choose File Quality',
+  mp3: 'MP3',
+  original: 'Original',
+  downloadAll: 'Download All'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  root: {
+    borderWidth: 1,
+    borderColor: palette.borderDefault,
+    borderRadius: spacing(2),
+    marginBottom: spacing(2)
+  },
+  titleContainer: {
+    padding: spacing(4),
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  titleContainerInner: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: spacing(2)
+  },
+  title: {
+    letterSpacing: 0.5
```

Comment:
Do we not have a Text component on mobile yet that let's us do harmony text variants?